frontend/server/src/DAO/Contests.php

```diff
@@ -652,6 +655,12 @@ final public static function getNextRegisteredContestForUser(
         $futureCondition = \OmegaUp\DAO\Enum\ActiveStatus::sql(
             \OmegaUp\DAO\Enum\ActiveStatus::FUTURE
         );
+        if (is_null($dayLimit)) {
+            $withinDayLimitCondition = true;
+        } else {
+            $withinDayLimitCondition = "DATEDIFF(start_time, NOW()) < $dayLimit";
+        }
```

Comment:
¿Acá podrás asignar `true` como cadena?

Esto para no revolver tipos de datos en la misma variable.

De paso, puedes simplificar un poco el código, para que sea más legible
```suggestion
        $withinDayLimitCondition = 'TRUE';
        if (!is_null($dayLimit)) {
            $withinDayLimitCondition = "DATEDIFF(start_time, NOW()) < $dayLimit";
        }
```